build.sh

```diff
@@ -20,89 +21,89 @@ fi
   popd
 
 # First off, we build the base php 8 image.
 pushd alpine-base-php/php8 && \
   make VERSION=${VERSION8} EXTRAVERSION=${EXTRAVERSION} UPSTREAM=${BASE} build && \
-  docker tag unocha/base-php:${VERSION8}${EXTRAVERSION} unocha/base-php:8.0-develop && \
+  docker tag unocha/base-php:${VERSION8}${EXTRAVERSION} unocha/base-php:8.0-${STABILITY} && \
   popd
 
 # Build the base php 8 image with NewRelic.
 pushd alpine-base-php/php8-newrelic && \
   make VERSION=${VERSION8} EXTRAVERSION=-NR${EXTRAVERSION} UPSTREAM=${VERSION8}${EXTRAVERSION} build && \
-  docker tag unocha/base-php:${VERSION8}-NR${EXTRAVERSION} unocha/base-php:8.0-NR-develop && \
+  docker tag unocha/base-php:${VERSION8}-NR${EXTRAVERSION} unocha/base-php:8.0-NR-${STABILITY} && \
   popd
 
 # Build the standard php 8 image.
 pushd alpine-php/php8 && \
   make VERSION=${VERSION8} EXTRAVERSION=${EXTRAVERSION} UPSTREAM=${VERSION8}${EXTRAVERSION} build && \
-  docker tag unocha/php:${VERSION8}${EXTRAVERSION} unocha/php:8.0-develop && \
+  docker tag unocha/php:${VERSION8}${EXTRAVERSION} unocha/php:8.0-${STABILITY} && \
   docker tag unocha/php:${VERSION8}${EXTRAVERSION} public.ecr.aws/unocha/php:${VERSION8}${EXTRAVERSION} && \
-  docker tag unocha/php:${VERSION8}${EXTRAVERSION} public.ecr.aws/unocha/php:8.0-develop && \
+  docker tag unocha/php:${VERSION8}${EXTRAVERSION} public.ecr.aws/unocha/php:8.0-${STABILITY} && \
   popd
 
 # Build the standard php 8 image with New Relic.
 pushd alpine-php/php8 && \
   make VERSION=${VERSION8} EXTRAVERSION=-NR${EXTRAVERSION} UPSTREAM=${VERSION8}-NR${EXTRAVERSION} build && \
-  docker tag unocha/php:${VERSION8}-NR${EXTRAVERSION} unocha/php:8.0-NR-develop && \
+  docker tag unocha/php:${VERSION8}-NR${EXTRAVERSION} unocha/php:8.0-NR-${STABILITY} && \
   docker tag unocha/php:${VERSION8}-NR${EXTRAVERSION} public.ecr.aws/unocha/php:${VERSION8}-NR${EXTRAVERSION} && \
-  docker tag unocha/php:${VERSION8}-NR${EXTRAVERSION} public.ecr.aws/unocha/php:8.0-NR-develop && \
+  docker tag unocha/php:${VERSION8}-NR${EXTRAVERSION} public.ecr.aws/unocha/php:8.0-NR-${STABILITY} && \
   popd
 
 # Build the k8s php 8 image.
 pushd alpine-php/php-k8s-v8 && \
   make VERSION=${VERSION8} EXTRAVERSION=${EXTRAVERSION} UPSTREAM=${VERSION8}${EXTRAVERSION} build && \
-  docker tag unocha/base-php:${VERSION8}${EXTRAVERSION} unocha/php-k8s:8.0-develop && \
-  docker tag unocha/base-php:${VERSION8}${EXTRAVERSION} public.ecr.aws/unocha/php-k8s:${VERSION8}${EXTRAVERSION} && \
-  docker tag unocha/base-php:${VERSION8}${EXTRAVERSION} public.ecr.aws/unocha/php-k8s:8.0-develop && \
+  docker tag unocha/php-k8s:${VERSION8}${EXTRAVERSION} unocha/php-k8s:8.0-${STABILITY} && \
+  docker tag unocha/php-k8s:${VERSION8}${EXTRAVERSION} public.ecr.aws/unocha/php-k8s:${VERSION8}${EXTRAVERSION} && \
+  docker tag unocha/php-k8s:${VERSION8}${EXTRAVERSION} public.ecr.aws/unocha/php-k8s:8.0-${STABILITY} && \
```

Comment:
This is the important bit.

Reply:
composer 2 and php7 from main instead of edge is also very nice :)